openapi.yaml

```diff
@@ -0,0 +1,69 @@
+swagger: '2.0'
+info:
+  version: '1.0.0'
+  title: Fulcio
+schemes: [http, https]
+host: fulcio.sigstore.dev
+basePath: /v1
+consumes: [application/json]
+produces: [application/json]
+securityDefinitions:
+  Bearer:
+    type: apiKey
+    name: Authorization
+    in: header
+paths:
+  /submitcsr:
+    post:
+      description: 'Request API for signing certifcate'
+      operationId: SubmitCSR
+      consumes:
+        - "application/json"
+      parameters:
+        - name: 'submitcsr'
+          in: 'body'
+          required: true
+          description: 'Submit CSR JWT Payload'
+          schema:
+            $ref: '#/definitions/Submit'
+      responses:
+        200:
+          description: Successful CSR Submit
+          schema:
+            $ref: '#/definitions/SubmitSuccess'
+        400:
+          description: Bad Request
+        404:
+          schema:
+            type: string
+          description: Invalid Data
+        500:
+          schema:
+            type: string
+          description: Server error
+definitions:
+  Submit:
+    type: object
+    required: [email,common_name,country,province,organization,organization_unit]
```

Comment:
Not sure if this should be outlined here, as I expect the input would be a single JWT base64 encoded. We would then verify the signing and decode to then serialise the values from there.

Reply:
Since you'll have a signed JWT, I don't think it makes sense to specify the schema here. I would just have a single property:

```
type: string
format: byte
```